code/fireball/fireballs.cpp

```diff
@@ -407,10 +407,15 @@ void fireball_parse_tbl()
 	{
 		if (fi.lod_count > 1)
 		{
+			Assertion(fi.lod_count < MAX_FIREBALL_LOD, "Fireball LOD (%d) greater than MAX_FIREBALL_LOD %d.", fi.lod_count, MAX_FIREBALL_LOD);
+
 			auto lod0 = fi.lod[0].filename;
 
+			if (strlen(lod0) > MAX_FIREBALL_BASENAME_LEN)
+				Error(__FILE__, __LINE__, "A fireball base file name may not be longer than %d characters due to the LOD naming scheme. (is %s)", MAX_FIREBALL_BASENAME_LEN, lod0);
```

Comment:
This should be `Error(LOCATION` to be consistent with the rest of the code.  And in fact, rather than making this an `Error`, change it to a `Warning` and just skip past the invalid LOD.

Reply:
Will do. I'd like to point out, however, that in these cases there can be no single invalid LOD, but all LODs except LOD0 will be skipped, due to having too long file names.

Reply:
Good point.  Even so, I think a Warning is still appropriate, because even though the situation is not desirable, it is still recoverable.